Administrator Guide/index.md

```diff
@@ -9,34 +9,38 @@
 	*  [Architecture](../Quick-Start-Guide/Architecture.md)
 	*  [Glossary](../glossary.md)
 	*  [Other Features of GlusterFS](./Did You Know.md)
-	*  [Setting Up Trusted Storage Pools](./Storage Pools.md)
-	*  [Managing the Gluster Service](./Start Stop Daemon.md)
 	
-3.  [POSIX Access Control Lists](./Access Control Lists.md)
+3.  Getting Started
 
-4.  [Accessing Data - Setting Up Clients](./Setting Up Clients.md)
+	*  [Managing the Gluster Service](./Start Stop Daemon.md)
+	*  [Setting Up Trusted Storage Pools](./Storage Pools.md)
+       
+4. Setting Up Storage
+
+	*  [Brick Naming Conventions](./Brick Naming Conventions.md)
+	*  [Formatting and Mounting Bricks](./formatting-and-mounting-bricks.md)
+	*  [POSIX Access Control Lists](./Access Control Lists.md)
```

Comment:
Isn't this a client side operation ? IOW, it needs a mounted volume.

Reply:
The page actually talks abt mounting the brick with -o acl .